### ISSUE TITLE
Object Ambient Lighting Issues

### ISSUE THREAD

**Issue by Jiroga** [Sulayre/WebfishingLure#3]
When installing Lure and Nubz4lif's Day/Night cycle, the ambient lighting of objects _partially_ works.

This issue has been present since v2.2.4 where no ambient lighting would emit from objects, but as of v3.0.0 this issue has been alleviated some, but not completely. Currently, some of the affected objects/locations are:
- Vending Machine: _Works as of 3.0.0_
- Bouncing Mushrooms: _Broken since 2.2.4_
- Campfire: _Works as of 3.0.0_
- Building Interior: _Broken since 2.2.4_

Using additional versions for testing, 2.2.3 was skipped due to possible data stability issues, but v2.2.2 and lower (tested down to 2.2.0) shows no issues in regards to object ambient lighting.

I also want to take a small opportunity to express my gratitude for all of your hard and quick work! You have a wonderful platform for modders to take off from. Keep up the fantastic work, Sulayre. 👍

**Comment by Sulayre** [comment]
seems like a day/night cycle issue because i have not touched the environment code in the slightest, may be godot's weird behavior when handling pcks, gonna see if i can come up with a possible fix next patch but for now i have no idea, thank you for reporting !

**Comment by Jiroga** [comment]
Good news! After a bit of in-depth testing, I can safely say that after 3.1.0 this bug is squished! ... sort of.

I noticed that after the addition of any map (such as the test map in v3.0.1 or Zea's WB_Construct map), the lightsources completely fail to function. However, I am entirely unsure if this issue may be due to the implementation of the other maps, or Day/Night Cycle not working nicely with more than one map being loaded.

In either case, it could be worth looking into it if you feel it necessary, otherwise we could say this issue has been resolved! :>

(PS: For the sake of posterity, I'm attaching the logs both without and with the additional map in the case you might find anything of use in them.)
[godot-nomap.log](https://github.com/user-attachments/files/17564174/godot-nomap.log)
[godot-withmap.log](https://github.com/user-attachments/files/17564176/godot-withmap.log)
